sdk/network/Azure.ResourceManager.Network/src/Generated/Models/ConnectivityParameters.cs

```diff
@@ -38,7 +38,19 @@ public ConnectivityParameters(ConnectivitySource source, ConnectivityDestination
         /// <summary> Network protocol. </summary>
         public Protocol? Protocol { get; set; }
         /// <summary> Configuration of the protocol. </summary>
-        public ProtocolConfiguration ProtocolConfiguration { get; set; }
+        internal ProtocolConfiguration ProtocolConfiguration { get; set; }
+        /// <summary> HTTP configuration of the connectivity check. </summary>
+        public HttpConfiguration ProtocolHttpConfiguration
```

Comment:
maybe `HttpProtocolConfiguration` is better?

Reply:
resolved with directive